helm-charts/helm-repo/ibm_cloud_pak/manifest.yaml

```diff
@@ -1,22 +1,5 @@
 images:
-- image: ibm-helm-repo-operator:3.6.2
-  references:
-  - repository: ibm-helm-repo-operator:3.6.2
-    pull-repository: quay.io/opencloudio/ibm-helm-repo-operator:3.6.2
```

Comment:
I thought both the tags and digest values were needed

Reply:
only digest is needed for airgap. In any case,  we actually have replaced the manifest.yaml with resources.yaml in case bundle. This file is kept here just in case the content linter complains about it. It has no real usage